readthedocs/templates/core/project_bar_base.html

```diff
@@ -21,6 +21,9 @@ <h1>
           {% endblock %}
         </span>
         <a href="{{ project.get_absolute_url }}">{{ project }}</a>
+        {% if project.delisted %}
+          <span class="quiet">{% trans "delisted" %}</span>
```

Comment:
This isn't a great user-facing message. We definitely need more here, with at least a link to the docs?

Reply:
I've added a link.

According to our own policy, we are also supposed to inform project owners via email.

Reply:
Code should probably handle that automatically, in an updated PR. Seems really obvious & easy to automate, so we don't miss it.